Scripts/slack.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+# usage:
+#  - with default arguments:
+#      slack.sh
+#  - with custom arguments:
+#      slack.sh [command] [arguments...]
+#      command and arguments: same as in slack channel after "/cci"
+
+if [ "$#" -eq 0 ]; then
+# if there are no args, use default coomand
+    SALCK_COMMAND='deploy'
+    SALCK_TEXT='deploy fourd alpha emails:gab.horv@gmail.com unofficial_release:true customChangelog:"Test changelog with spaces" feature/4DM-5808-Update-deploy-from-custom-branch'
```

Comment:
Double "deploy"?

Reply:
Yes, the project needs a 'command' field, but - as far as i see - it doesn't use it, it uses the first part of 'text' field as command instead (but it throws an error if command field is missing).